src/bin/contract_monitor/main.rs

```diff
@@ -1,5 +1,6 @@
 //! Main entry point for ContractMonitor
 
+#![allow(dead_code)]
```

Comment:
Do we want to allow dead code, or should we delete dead code?

Reply:
I mentioned in the description that it seemed like the dead code was methods that are not yet but will be used in the future. I'm open to deleting them but wasn't sure.